ee_tests/Dockerfile.builder

```diff
@@ -1,13 +1,14 @@
 FROM centos:7
 ENV LANG=en_US.utf8
-#ENV NPM_CONFIG_LOGLEVEL info
 ENV NODE_VERSION 6.11.4
 ENV DISPLAY=:99
 
 ENTRYPOINT ["/opt/fabric8-test/docker-entrypoint.sh"]
+WORKDIR /opt/fabric8-test/
```

Comment:
Move WORKDIR one line up and replace all occurences of `/opt/fabric8-test/` with `./`